features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/ConvertToEvent.java

```diff
@@ -73,383 +73,468 @@
  */
 public class ConvertToEvent {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ConvertToEvent.class);
```

Comment:
This diff is too large because the whitespace was reformatted and the text was rewrapped. Can you undo the formatting changes so that it's easier to see the changed lines of code?